constructor-manager-cli/setup.cfg

```diff
@@ -0,0 +1,59 @@
+[metadata]
+name = constructor-manager-cli
+version = 0.0.1
+description = Constructor environment and updates manager command line interface
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/napari/packaging/constructor-manager-cli
+author = napari team
+author_email = napari-steering-council@googlegroups.com
+license = MIT
+license_files = LICENSE
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Topic :: Scientific/Engineering :: Image Processing
+project_urls =
+    Bug Tracker = https://github.com/napari/packaging/issues
+    Source Code = https://github.com/napari/packaging/constructor-manager-cli
+
+[options]
+packages = find:
+install_requires =
+    packaging
+    requests
+    pyyaml
+python_requires = >=3.8
+include_package_data = True
+package_dir =
+    =src
```

Comment:
is this a typo or intended?

Reply:
Intended the foldert structure for the pakcages is

`package-name/src/module_name`

Reply:
I think @potating-potato was referring to the `=src`. The syntax in setup.cfg here is confusing (bad) but this is not a typo. This gets parsed into a dictionary and that line will have an empty key: https://setuptools.pypa.io/en/latest/userguide/declarative_config.html#specifying-values

Reply:
Ahhh got it, yes the syntax is weird. We can eventually migrate to a pyproject toml only and use hatch as the build system https://github.com/pypa/hatch

Reply:
thanks for confirming! I indeed never seen this syntax before